code/scripts/mock/db-seed.js

```diff
@@ -1,34 +1,41 @@
 // code/scripts/mock/db-seed.js
-import { prisma } from "../../src/lib/prisma.js";
+import {
```

Comment:
Maybe time to split this seed file into multiple files

Reply:
Good suggestion. The updated mock data are split into multiple files w/ some prefix like ```class-list```.